data/affycoretools/affycoretools.json

```diff
@@ -145,6 +145,10 @@
                 "title": "Orchestrating high-throughput genomic analysis with Bioconductor"
             },
             "type": "Other"
+        },
+        {
+            "doi": "10.1038/nmeth.3252",
```

Comment:
This citation is a bioconductor citation and should not be added. Should we remove this citation altogether from the repo?

Reply:
ping @hmenager